app/lib/bk/compat/parsers/bitbucket/steps.rb

```diff
@@ -29,6 +29,7 @@ def translator(conf, *, **)
             agents: translate_agents(step.slice('size', 'runs-on'))
           ).tap do |cmd|
             cmd.timeout_in_minutes = step.fetch('max-time', nil)
+            translate_after_script(cmd) if !step['after-script'].nil?
```

Comment:
I would inline the `cmd.add_commands` call here as it is a single line

Reply:
Switched 👍